Makefile

```diff
@@ -123,7 +123,11 @@ dll
 {%-   endcapture %}
 build-{{build_target}}-{{target_type}}:
 	cargo build --target {{build_target}} {{build_arg}}
+{%-   if export_target == "Windows Desktop" %}
```

Comment:
Doesn't this condition seem inverted here? Also I think you might want to check the host instead of target platform here, using the `os-arch` variable documented here: https://github.com/cargo-generate/cargo-generate#templates

Reply:
> Doesn't this condition seem inverted here?

Oops. Fixed.

> Also I think you might want to check the host instead of target platform here.

This actually confuses me a little bit. I think this boils down to cross compilation? Essentially I have no clue if cross compilation is possible, like building the `build-i686-unknown-linux-gnu-debug` target from a Windows machine? If so, I would indeed need to use the current OS rather than the target OS. Right now I assumed one would only make the Windows targets from a Windows system.

Reply:
Another thing is that the template might be generated on one system, versioned and pushed on a repository, and clone on another system. In such a case, do we want to tie the Makefile to the system that instantiated the template?

Reply:
> Essentially I have no clue if cross compilation is possible

It should be as long as you have the appropriate platform C headers and toolchain installed.

> In such a case, do we want to tie the Makefile to the system that instantiated the template?

Ideally we don't. [`project-utils`](https://github.com/godot-rust/project-utils) handles cross-platform paths properly, but that's not integrated for now (see https://github.com/godot-rust/godot-rust-template/pull/40#issuecomment-812879758). Personally I think it's a fine temporary workaround. Note that [`run` already depends on the host system arch](https://github.com/godot-rust/godot-rust-template/blob/48d3db8040e185dcdbed613b2775fa6059d9d5a7/Makefile#L154).

Reply:
Thanks for the pointers!

> Note that [`run` already depends on the host system arch](https://github.com/godot-rust/godot-rust-template/blob/48d3db8040e185dcdbed613b2775fa6059d9d5a7/Makefile#L154).

I hadn't noticed, and I'm not really fond of it... I'll leave the issue of using `project-utils` alone, even though I agree with you that this repository being a template should thrive to reuse other good projects for such utilities. At least that is what I expected out of it when using it...

> Personally I think it's a fine temporary workaround.

Are you talking about my current changes, or using `os-arch` instead?

Reply:
I'd prefer using `os-arch` if just to be consistent with `run`, although I don't have a strong opinion about it. If another committer thinks that the current changes are good, they may merge the PR.

Reply:
At least this could remove the dependency on coreutils, this could work for now.

Reply:
@toasteater I now use `os-arch` to determine what command to spawn from the Makefile.

Reply:
Well... I removed this conditional branching completely, see https://github.com/godot-rust/godot-rust-template/pull/40#issuecomment-813756454